tests/core/gui/test_recovery.py

```diff
@@ -72,7 +72,9 @@ async def test_export_recovery_device(
     assert exp_w.current_page.combo_devices.count() == 1
     assert exp_w.current_page.combo_devices.currentData() == alice.slug
 
-    exp_w.current_page.label_file_path.setText(str(tmp_path))
+    exp_w.current_page.label_file_path.setText(
+        str(tmp_path) + "/" + get_recovery_device_file_name(alice)
```

Comment:
`str(tmp_path / get_recovery_device_file_name(alice))` (assuming that `tmp_path` is some kind of `Path`)

`+` to concatenate should probably never be used.